tests/common/helpers/assertions.py

```diff
@@ -9,6 +9,16 @@ def pytest_assert(condition, message=None):
         pytest.fail(message)
 
 
+def pytest_expect(condition, message=None):
```

Comment:
>pytest_expect [](http://example.com/codeflow?start=4&length=13)

use try catch finally instead of this way.

Reply:
Done.